packages/odie-client/src/assets/human-avatar.tsx

```diff
@@ -0,0 +1,28 @@
Cstd23n/a3caQV0QqUs9j2EukgTdMpqjcPxTa3YtG+9lVKSBByEWep7d8RuvwQPHkNclVgpg003j9RK16Sf0t9DXiL3gdvYOq6GDw54VTtDBeuY3sDOO7ptwYPDBjVpnWk2TtWlkEcLGprmWL11lmrw8K0pklNdLmBFl9KpAau6nO/JqT2sJQ8Fx5i89mr0MXdlOuj4UBxuB+CapnLqicXTX5ptXdi0m0Ufqy1+rbL0GRrGb7ObKY+HdXNprfI05nS1fL0KYs6sVT65bh7xlCNtdgNBPLqH3kCSkdsd5p63PXoTRwSpiJ17T899w2atqn6+4rsxjaR2KTr7Vbf4pK+IbDe1zIMQNxvp0tO17O+rnReDbAXEm1PxzMei5U+7ZlPv6SpyXzHGdLBNfDBaHuTxv/X3XLLRwwMWiVF3btQ7rzX83b4/1qk/4hbPS8vADhP4vOBjZL3T6Py+kVwB1as+0pu55OfqUtk6gGmXsLgq8QLJAq4+aqIPF97jHJJhbKxJHSbTNR8gehdMCd7jM3TmjIuATDWsETHIPjiSorwmbRCfJJsvxiiMAmMYZCyC0mDIpJpSzf1QtUuDwSMgAa6iYpoMvnd/BOUJ3B5BF98iq5ejKYZN93gmRgvFTIh9wiN+GJv3ZgLXYrq4NtbgqiMxC6ajuUEhHt1Hr6MqOD2SXfHI1cWMBnnzZA/TRfWB4ZfrTYkuIShLEGJ6I6lgnxJdwKuiIzgX6qEXVk+JLiGFUOclZENP4E6JHoCAoCNMGzXs9oYp0SU2K9gzTEQQovK+riFFUMdAtAen8DKEe9Pned6hzthD0EmZwB2epj5cO9pDPoLXxs4bZzQBst8hgd7jPTrzkqWZV1S0BMORPcnbMuq4Oq46HIk+VKK9xycEnbu19Vesq0bHHvMn5qjt50JyllVK5c5lvX491ovZ4llOHCvCF/tfYvG3SglCfJ1U44J9BVK24hCJzouhoszdeiW7+a26j6vLX9mofD0NGkGea9n1lXiPE0xGY+OaXPs3dfel5txT8exX2/4JP5RnQ+Sh6hATYKvDyb8cLtGCd07Sdd1/Oei9YtSatE53q59O6hUCFLviOs/LqIzaxs3qje8lb/2zi/9YnZl1ZPlUlOGoa0El6C6mhasNoaxJLzz5D0NZ2M7w4PEida3nWfPR79P42oLpB+pkl7Neq/z23DYe7Yv8rAed0Vi/Wbn+g+RP37K91zDz0eI3e/OftrXqEJ9el8etvMJV8JWDfoWHSHQuE0ZUMKuD5ZrPqunvs/BV27mh+j38jFXzNq9o0eXBgkjRiOcVTnAp9Z6d77xUufHv0Z/+JQtfRr/Xnvxm/+w30rmTaqhFGsUX7/ABvobXR4fovKQzExXpU83K8brP2u2r8foLQedl10+VMzWtNUJmfKJwZe42RVKCNKmnG3LnevDWc7Mr/6pWvt3vXiY4Yxefzc79HcvnVX3Y4c2SaFG4SlkBsl/tcdhhUgFlcA1uq49GzNb9vHvrO+3wsl/9x7n+c63OR6LWJ9rm0UgtUD2OqeE9aU8nbzf9W8fdb3trz23cfnHNrs0Qzsx+bOPEs+l7/kpOnpfaqIp+pGw0sTXsAYR6HBkW0JqoVrEn3u8qzybBab/6kxPpi9Xez3vdF5LqT60669WcC2bQVbyTrO+ydurXev6qCv/QdGT1J9P5z9qFL6QnPuuPL0qeEh8N0YDE6Da6iQsGCNvj6caQM8xRPNXPYtejmfbrJ+Pnems/vn37t9r2KvQViZBK0TCvHUFKNaVW082Z1vtYfqa78MVw5r2+gZgDZGp2CYszJPMki9iZ0vzY40nHlmGR8pErcrwezzy1mpxLm1+gebnSv3zMXW3KStWva/qgUlqRzLfd8oaci2Yu2YULHF9MK3NOGO7ut81CixmGQRfbwNXx++JsnKmsIu9nsIHqVOYlmDczT7p0te/eRjYSH2pSkIxaIq3UzwkLWe141EKq4NGHNvU1LxdOCLq4BtnMfp50NAE5Q484NFDBVVSf5ZjljQccBWUM7FDHDW2y6dBdTBVXxdaPjurYAilr0nJ9slUHSvmzEKKx7Cl5WVN/YFfcY7n+ZBC9ibIGYTL7wfJked6Z4Rp7SwxMA/97gCiURXfQ3TKBu2ump0TvAXmToI6KUV57mksxJXov2EzgdosZhrt/xveU6D0iN6vzDty9zDCcEr0XDI4gi8p5uOwqgTsleu/IFUhGkCdw7a6MvCnR+4IgDhNhwrKFfyrRw8dmsa8jCAm6iN25KHJK9H4hQNHaraIyKrA911OiDwTJ0CEm7+5+qKaeEn0gCKgU3dn5wQJqwh8xNelQiMP0tnYf36dAPN6hnBvnNLijjrwCQqUEXXT3wXPV8mNMJcA7bB4zm8yg2SRDihhvvivaGm5gVkIu38agBbW8SKtZlLJNsR/kXCeYENND2dJRlILSmRpLc6gPPsrSAs5jj+6w1zFi06YGFROUu2KuQJzDOpZm+OAy6pMf4NwpRArtMcV+UCZwdQe9gfSLitac0vPHePoM6qlHOLdEpQrTHfEgyHNdWx9C5yAwnJ/l40uoMws8cZH3XcQY+vG4l3tEMaBATL943nCUEhieWOKpJS4ewzQqfPxxVu6ycpvL16kGqKkTsz9sCnWbtEHW4MIyf/N+nj5Dy2CAhWN87kNcu0U/4a1VjKFW3eEzp3gwBCxJh1hx7hJfvsRfXmCpDptZ8LOL/PVnUMJPfsGN2/RjELRCa2RqX+8Ij/PY3B1xVKqcPc7nH+Mrj3NupjikINpoPnCO403OLvEfP+X1KyRZOTI0V0BTrreDx3vEo0ApKnXed4FnPsXnnuLMApXyKPFbHZXVDV54jd9c4eoqV29x+w5hj8wiw25Cf5cgj2MoWg1OznPuFOdP8YELfPQxTs5tOfD+ApqTs3zuw1w8w6/+gP4N/Zh+TJrBVIE8CEULnqLV5OJZ/vxJPnKJ9y5Sr9x/5P8DMo5/sshpyMoAAAAASUVORK5CYII="
```

Comment:
This image should be extracted to a `.png` file. What if I want to see how the PNG image looks like?

Webpack has capabilities (with `file-loader` and `url-loader`) to import such a file and convert it into an inline data URL. The output of the build can be identical to the current state, but the source can be managed in a much nicer way where JavaScript is in `.js` files and images are in `.png` files.

Reply:
Thanks for pointing this out @jsnajdr I honestly didn't even inspect the SVG much. This was copied directly from the Figma. We have a follow-up PR to work on the stylings and will take this into account and address this.